sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -245,9 +255,11 @@ public void queryPlanDiagnostics() {
     }
 
     @Test(groups = {"simple"}, dataProvider = "query", timeOut = TIMEOUT)
-    public void queryMetrics(String query) {
+    public void queryMetrics(String query, Boolean qmEnabled) {
```

Comment:
why Boolean vs boolean, here and other place ?

Reply:
To be able to test default (not explicity set)